sim/src/main.rs

```diff
@@ -9,33 +9,35 @@ mod traffic_generator;
 
 use channel::Channel;
 use link::Link;
-use plugin_wrapper::PluginWrapper;
 use simulator::Simulator;
 use traffic_generator::TrafficGenerator;
-use std::borrow::BorrowMut;
 use std::path::PathBuf;
 
-static COMPILED: &str = "target/debug/libfilter_lib";
+//static COMPILED: &str = "../target/debug/libfilter_lib";
+static COMPILED: &str = "../target/debug/librust_lib.dylib";
```

Comment:
dylib is only supported on Mac. Use the facilities I wrote for cross-platform compatibility.

Reply:
Sorry, didn't realize dylib was only Mac;  I thought your stuff just consisted of the CARGO_MANIFEST_DIR stuff.  Fixed now.